api/users/views.py

```diff
@@ -18,11 +21,13 @@ class UserMixin(object):
 
     def get_user(self, check_permissions=True):
         obj = get_object_or_404(User, self.kwargs[self.node_lookup_url_kwarg])
-        if check_permissions:
+
+        if check_permissions & obj.is_disabled is False:
```

Comment:
It *should* be good to go. Here's the current code for it:

```python
    @property
    def is_active(self):
        """Returns True if the user is active. The user must have activated
        their account, must not be deleted, suspended, etc.
        :return: bool
        """
        return (self.is_registered and
                self.password is not None and
                not self.is_merged and
                not self.is_disabled and
                self.is_confirmed)
```

A quick glance shows that it also catches the case where the user has been merged into another, which I don't believe is caught in this PR.